### DISCUSSION TITLE
[Bug_Markdown] Copilot's prompt cannot be quickly confirmed via Tab

### DISCUSSION THREAD

**Discussion by BeiningYing** [community/community#55616] [Copilot Conversations]
### Select Topic Area

Bug

### Body

In the VScode editor environment, editing a block of code within [Markdown], Copilot's prompt cannot be quickly confirmed via [Tab].
The image below：
![image](https://github.com/community/community/assets/118277279/3ff38fb4-1760-4cf3-84c9-5f00f5d12bc2)

This problem is very bad for operational efficiency.
Is there a conflict with Markdown's own Tab shortcut? It doesn't seem to be?

**Comment by Akash1134** [comment]
Hi @BeiningYing , 

Would it be possible for you to record a short video reproducing the issue and [sharing the logs](https://docs.github.com/en/enterprise-cloud@latest/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-your-environment?tool=vscode#viewing-and-collecting-log-files)?

Thanks!
